docs/_v9_1/4_logging.md

```diff
@@ -18,7 +18,7 @@ There are numerous ways to implement a logger, but the following examples will b
 
 1. Install the Nuget-packages `NLog`, `NLog.Extensions.Logging` and `Microsoft.Extensions.DependencyInjection`.
 1. Create an `nlog.config` file. The following is an example that logs to both file and console:
-``` xml
+{% highlight xml %}
```

Comment:
Er det en spesiell grunn til at akkurat dette eksemplet har brukt annen highlightsyntaks enn de andre?

Reply:
Selv om XML er inne en `````` slik codeblock, så vil den ikke blir "escaped". Og det førte til masse problemer. Så vi må bruke en annen codeblock akkurat for XML ting.

Reply:
Right. Ship it